### PR TITLE
python3Packages.click-aliases: 1.0.1 -> 1.0.2

### PR DESCRIPTION
**Pull request by panicgh**
## Description of changes

Upstream doesn't provide a changelog, but the changes were quite small:
https://github.com/click-contrib/click-aliases/compare/v1.0.1...v1.0.2

Backporting because this is a patch release.

~I also added `passthru.updateScript`.~

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

----

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>pynitrokey</li>
    <li>pynitrokey.dist</li>
    <li>python310Packages.click-aliases</li>
    <li>python310Packages.click-aliases.dist</li>
    <li>python311Packages.click-aliases</li>
    <li>python311Packages.click-aliases.dist</li>
  </ul>
</details>

### CODE REVIEW

**Comment by natsukium** [comment]
@ofborg build python311Packages.click-aliases python310Packages.click-aliases

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 257787` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>pynitrokey</li>
    <li>pynitrokey.dist</li>
    <li>python310Packages.click-aliases</li>
    <li>python310Packages.click-aliases.dist</li>
    <li>python311Packages.click-aliases</li>
    <li>python311Packages.click-aliases.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/click-aliases/default.nix [hunk 1]
```diff
@@ -30,6 +27,8 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "click_aliases" ];
 
+  passthru.updateScript = nix-update-script { };
+
```
Comment (fabaff) [inline pkgs/development/python-modules/click-aliases/default.nix:31]: This is not needed. `nix-update` works usually fine on Python modules.

```suggestion

```

Comment (panicgh) [reply pkgs/development/python-modules/click-aliases/default.nix:31]: I just tried `nix-shell ./maintainers/scripts/update.nix --argstr package python3Packages.click-aliases` on `master` and it did not work.

**Review by mweinelt — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/click-aliases/default.nix [hunk 2]
```diff
@@ -3,23 +3,20 @@
 , fetchFromGitHub
 , click
 , pytestCheckHook
+, nix-update-script
 }:
 
 buildPythonPackage rec {
   pname = "click-aliases";
-  version = "1.0.1";
+  version = "1.0.2";
```
[the remark this answers is not in the document]
Comment (panicgh) [reply pkgs/development/python-modules/click-aliases/default.nix:12]: Thanks for the suggestion, I was unaware of this attribute.

File: pkgs/development/python-modules/click-aliases/default.nix (hunk 1 above)
Comment (panicgh) [reply pkgs/development/python-modules/click-aliases/default.nix]: I removed the updateScript as requested, even though updating the package then doesn't work automatically.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**